site/content/en/docs/administration/basics/installation.md

```diff
@@ -503,3 +507,64 @@ If you stay in China, for installation you need to override the following source
   ```bash
   curl https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
   ```
+
+### HTTPS is not working because of a certificate
+
+If you're having trouble with SSL connection, you need to get logs traefic.
+First, get the traefik container ID from the docker:
+
+```bash
+docker ps --filter NAME=^traefik$
```

Comment:
In our case the container always has `traefik`name, so `docker logs traefik` works